prawcore/rate_limit.py

```diff
@@ -38,10 +41,12 @@ def delay(self):
         """Sleep for an amount of time to remain under the rate limit."""
         if self.next_request_timestamp is None:
             return
-        sleep_seconds = self.next_request_timestamp - time.time()
-        if sleep_seconds <= 0:
+        stime = self.next_request_timestamp - time.time()
+        if stime <= 0:
             return
-        time.sleep(sleep_seconds)
+        msg = 'Sleeping: {:0.2f} seconds prior to call'.format(stime)
```

Comment:
Can you either do away with this variable, or call it `message`? I'm a little pedantic about not shortening variable names.

Reply:
``message`` is fine.